src/config/mod.rs

```diff
@@ -3,6 +3,7 @@
 pub mod config_args;
 pub mod deserialized_config;
 pub use deserialized_config::*;
+pub mod wizard;
```

Comment:
Did you check if any wizard does not already exist before creating your own? Sth like https://crates.io/crates/requestty or https://crates.io/crates/enquirer or https://crates.io/crates/promptly?

Reply:
I didn't knew that this kind of crates exist! Thank you for pointing them out!